docs/src/pages/premium-themes/onepirate/modules/components/Paper.tsx

```diff
@@ -1,52 +1,42 @@
 import * as React from 'react';
-import clsx from 'clsx';
 import MuiPaper, { PaperProps } from '@material-ui/core/Paper';
-import { Theme } from '@material-ui/core/styles';
-import { withStyles, WithStyles } from '@material-ui/styles';
+import { experimentalStyled as styled } from '@material-ui/core/styles';
```

Comment:
```suggestion
import { styled } from '@material-ui/core/styles';
```

Reply:
I had not realized that he had already changed.